app/views/movements.scala.html

```diff
@@ -49,7 +49,7 @@
                     <td class="table-cell" id="submissionType-@submission.conversationId">@submission.submissionType</td>
                     <td class="table-cell" id="submissionAction-@submission.conversationId">@submission.submissionAction</td>
                     <td class="table-cell" id="dateUpdated-@submission.conversationId">
-                        @{notifications.headOption.map(n => n.dateTimeReceived).map(DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm").withZone(ZoneId.systemDefault()).format(_))}</td>
+                        @{notifications.headOption.map(n => n.timestampReceived).map(DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm").withZone(ZoneId.systemDefault()).format(_))}</td>
```

Comment:
(Not part of this PR change, but...) Would it not be useful to extract this date formatting logic somewhere so that it can be reused by other parts of the UI?

Reply:
Yeh Ideally we should pull it into a class which we can unit test and share across modules

Reply:
Also `n => n.timestampReceived` is the same as `_.timestampReceived`